src/Assets/updateScreen.ts

```diff
@@ -46,6 +46,7 @@ const getScreensBody = (formData: FormData, languageCode: Language, whiteLabel:
     has_erc: null,
     has_lifeline: formData.benefits.lifeline,
     has_leap: formData.benefits.leap,
+    has_ncci: formData.benefits.NCCrisisIntervention,
```

Comment:
Variables that start with a capital letter typically mean that the variable is a class or a React component.
```suggestion
    has_ncci: formData.benefits.ncCrisisIntervention,
```

Reply:
I have changes NCCrisisIntervention to nccip.